seatunnel-translation/seatunnel-translation-spark/seatunnel-translation-spark-common/src/main/java/org/apache/seatunnel/translation/spark/common/serialization/InternalRowConverter.java

```diff
@@ -196,6 +195,7 @@ private static Object reconvert(Object field, SeaTunnelDataType<?> dataType) {
                     default:
                         return arrayData.array();
                 }
+            case TIME:
```

Comment:
You need to `#reconvert` it to `LocalTime`

Reply:
LocalTime will cause the data Exception, we convert time->timestamp, writer to Sink, is timestamp, not time.